### DISCUSSION TITLE
Custom Samples on Blue Korg with Hacktribe

### DISCUSSION THREAD

**Discussion by dodofogel1005** [bangcorrupt/hacktribe#278] [General]
Hello,
Im totally new here,
I tried to Add custom Samples on my Electribe 2 Synth with Hacktribe. It shows me an error every time i try to save the samples on the electribe. I convert the files in 16bit wav. And add it into the e2sSample.all data. In the pogramm i usw it dont show the Factrory samples anyway. So the list is empty.

Sorry for my Bad english. Im from Germany.

I only want custom Samples on my Blue Korg. Thats the Reason why i updated to Hacktribe.

**Comment by bangcorrupt** [comment]
Does the error happen when you import individual samples on the electribe, or when you import the full e2sSample.all?

Are you using [Oe2sSLE](https://github.com/JonathanTaquet/Oe2sSLE) to edit e2sSample.all?

Can you load [this e2sSample.all](https://github.com/bangcorrupt/hacktribe/blob/main/content/carved_beats_set/e2sSample.all) without error?

<br/>

<br/>

Tritt der Fehler auf, wenn Sie einzelne Samples auf dem electribe importieren oder wenn Sie die vollständige Datei e2sSample.all importieren? 

Verwenden Sie [Oe2sSLE](https://github.com/JonathanTaquet/Oe2sSLE), um e2sSample.all zu bearbeiten? 

Können Sie dieses [e2sSample.all](https://github.com/bangcorrupt/hacktribe/blob/main/content/carved_beats_set/e2sSample.all)  ohne Fehler laden?


Entschuldigung für mein schlechtes Deutsch, ich habe maschinelle Übersetzung verwendet.

**Reply by dodofogel1005** [reply]
after sitting on it all night i made it.  I just overlooked that you have to start with user samples from number 501.  after i did that everything worked fine.
